both/api/sources/server/stream-chain/stream-types/transform-kobo/transform-kobo.ts

```diff
@@ -0,0 +1,370 @@
+    'properties.accessibility.restrooms.0.toilet.spaceOnUsersLeftSide':
+      parseFloatUnit(data, 'inside/toilet/free_space_front', usedLengthUnit),
+    'properties.accessibility.restrooms.0.toilet.spaceOnUsersRightSide':
+      parseFloatUnit(data, 'inside/toilet/free_space_left', usedLengthUnit),
+    'properties.accessibility.restrooms.0.toilet.spaceInFront':
+      parseFloatUnit(data, 'inside/toilet/free_space_right', usedLengthUnit),
+    // handles
+    'properties.accessibility.restrooms.0.toilet.hasFoldingHandles':
+      parseIsAnyOf(data, 'inside/toilet/has_arm_rests', ['left_and_right', 'right', 'left']),
+    'properties.accessibility.restrooms.0.toilet.foldingHandles':
+      parseIsAnyOfEntry(data, 'inside/toilet/has_arm_rests', ['left_and_right', 'right', 'left']),
+    'properties.accessibility.restrooms.0.toilet.foldingHandles.onUsersLeftSide':
+      parseIsAnyOf(data, 'inside/toilet/has_arm_rests', ['left_and_right', 'left']),
+    'properties.accessibility.restrooms.0.toilet.foldingHandles.onUsersRightSide':
+      parseIsAnyOf(data, 'inside/toilet/has_arm_rests', ['left_and_right', 'right']),
+    // washBasin
+    'properties.accessibility.restrooms.0.washBasin':
+      parseHasEntry(data, 'inside/toilet/has_basin'),
+    'properties.accessibility.restrooms.0.washBasin.accessibleWithWheelchair':
+      parseYesNo(data, 'inside/toilet/basin_wheelchair_reachable'),
+    'properties.accessibility.restrooms.0.washBasin.spaceBelow':
+      parseHasWithDefault(data,
+                          'inside/toilet/basin_wheelchair_fits_belows',
+                          wheelChairWashBasin,
+                          null),
+    'properties.accessibility.animalPolicy.allowsAnyAnimals':
+      parseYesNo(data, 'inquire/are_service_animals_allowed'),
+    'properties.accessibility.staff.isTrainedForDisabilities':
+      parseYesNo(data, 'inquire/staff_has_disabled_training'),
+    'properties.accessibility.staff.spokenLanguages':
+      parseMultiSelect(data, 'inquire/staff_spoken_sign_langs'),
+    'properties.accessibility.staff.isTrainedInSigning':
+      parseYesNo(data, 'inquire/staff_can_speak_sign_lang'),
+    'properties.accessibility.media.isLargePrint':
+      parseYesNo(data, 'inquire/media/has_large_print'),
+    'properties.accessibility.media.isAudio':
+      parseYesNo(data, 'inquire/media/has_audio'),
+    'properties.accessibility.media.isBraille':
+      parseYesNo(data, 'inquire/media/has_braille'),
+  };
+
+  const result = {};
+  // if there is a null in the history, do not set a value
+  const customizedSetter = (currValue: any) => currValue === null ? null : undefined;
+  for (const [key, value] of entries(mapping)) {
+    if (typeof value !== 'undefined') {
+      setWith(result, key, value, customizedSetter);
+    }
+  }
+
+  // rate place a11y
+  const a11y = evaluateWheelmapA11y(result);
+
+  // TODO these fields don't exist in ac format! Clarify & align with wheelmap frontend & ac-format
```

Comment:
I'd like to get rid of this and collapse it to a string-enum format again, as I think the decision to have the 2 booleans had bad consequences. This would be an extra task though. What do you think about this?

```
{
  simplifiedAccessibilityForProfile: {
    wheelchair: 'yes' | 'limited' | 'no',
    …
  }
}
```

https://trello.com/c/9iZd4ANV/1367-clean-up-personal-profile-in-ac-format-use-yes-no-limited-again

Reply:
👍